sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/metrics/_exporter.py

```diff
@@ -265,9 +268,4 @@ def _is_status_code_success(status_code: Optional[str], threshold: int) -> bool:
 def _get_metric_export_result(result: ExportResult) -> MetricExportResult:
     if result == ExportResult.SUCCESS:
         return MetricExportResult.SUCCESS
-    if result in (
-        ExportResult.FAILED_RETRYABLE,
-        ExportResult.FAILED_NOT_RETRYABLE,
-    ):
-        return MetricExportResult.FAILURE
-    return None
```

Comment:
What's the reason for this change?

Reply:
Not really necessary to return None anymore because there are no other cases besides the three `ExportResult`s